federation/pkg/dnsprovider/providers/coredns/rrsets.go

```diff
@@ -78,13 +77,10 @@ func (rrsets ResourceRecordSets) Get(name string) (dnsprovider.ResourceRecordSet
 		}
```

Comment:
CoreDNS does support TXT record.
Since we do not write/read any other records other than A and CNAME record, it should be fine for federation. We need to make it more generic further as DNS provider will be used by many others apart from federation.